apps/researcher/src/app/[locale]/research-guide/[id]/page.tsx

```diff
@@ -25,167 +33,303 @@ export default async function GuidePage({params}: Props) {
-                          {' — '}
-                          <span className="text-sm">
-                            <a target="_blank" href={citation.url}>
-                              {citation.url}
-                            </a>
-                          </span>
-                        </div>
-                      </li>
-                    ))}
-                  </ul>
-                </>
-              )}
+          <h1 className="text-4xl" tabIndex={0}>
+            {guide.name}
+          </h1>
+          {guide.alternateNames?.length ? (
+            <div className="text-consortium-purple-700">
+              <em>{t('nameVariations')}: </em>
+              {guide.alternateNames?.map((name, idx) => (
+                <span key={name}>
+                  {name}
+                  {idx < (guide.alternateNames?.length ?? 0) - 1 && (
+                    <span className="text-consortium-purple-300"> — </span>
+                  )}
+                </span>
+              ))}
             </div>
-          </div>
-          <div className="w-full md:w-1/3">
-            {guide.seeAlso && guide.seeAlso?.length > 0 && (
+          ) : null}
+          {guide.abstract && <p className="max-w-3xl">{guide.abstract}</p>}
+        </div>
+      </div>
+      <GuideNavigationBar links={navLinks} maxWidth="1500px" />
+      <main className="flex flex-col lg:flex-row gap-16 xl:gap-20 w-full max-w-[1500px] mx-auto px-4 sm:px-10  my-20">
+        <div className="w-full lg:w-2/3 xl:w-3/4">
+          <div className="prose !max-w-3xl prose-a:font-normal prose-a:decoration-neutral-400 prose-headings:scroll-mt-20">
+            {guide.text && <StringToMarkdown text={guide.text} />}
+            {guide.citations && guide.citations.length > 0 && (
               <>
-                <h2 className="mb-2" tabIndex={0}>
-                  {t('relatedItems')}
+                <h2 id="resources" className="scroll-mt-20" tabIndex={0}>
+                  {t('citations')}
                 </h2>
-                <div className="flex flex-col gap-2 mb-4">
-                  {guide.seeAlso?.map(item => (
-                    <Link
-                      key={item.id}
-                      href={`/research-guide/${encodeRouteSegment(item.id)}`}
-                      className="bg-consortium-sand-100 text-consortium-sand-800 no-underline hover:bg-consortium-sand-200 transition rounded flex flex-col p-2"
-                    >
-                      <div className="flex items-center justify-between gap-2">
-                        <div>{item.name}</div>
-                        <div>
-                          <ChevronRightIcon className="w-5 h-5 fill--consortium-sand-900" />
-                        </div>
-                      </div>
-                    </Link>
-                  ))}
-                </div>
+                <CitationList
+                  labelKey="primarySources"
+                  citations={guide.citations.filter(c => c.type === 'primary')}
```

Comment:
Or use the [keys from the enum](https://github.com/colonial-heritage/colonial-collections/blob/7028b39afede7905fdf30b6480daf7e4c6d8ae62/packages/api/src/research-guides/definitions.ts#L3-L6) instead?